mappings/net/minecraft/world/biome/layer/VanillaBiomeLayers.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS cez net/minecraft/world/biome/layer/BiomeLayers
+CLASS cez net/minecraft/world/biome/layer/VanillaBiomeLayers
```

Comment:
Why? You don't see us doing `VanillaBlocks`, etc.

Reply:
From https://github.com/FabricMC/yarn/pull/248/files#diff-db683513994329b7b6b9b89e7d6a283bR1

Reply:
@gegy1000 The names were garbage and mostly meant for understanding; I'd rather you use them as inspiration and not directly copy them in.

Do BiomeLayers... I don't remember why I did that anymore

Reply:
The class constructs the layer stack for the overworld -- possibly `OverworldLayerBuilder` or something similar?

Reply:
Yes, that's what i was going for - I wanted to emphasize it's specific to the overworld, not other dimensions... but maybe that's silly

Reply:
Went for `OverworldBiomeLayers` as `OverworldBiomeLayerBuilder` might be too long.

Reply:
The only utilities it has are specifically relating to overworld biomes, and only used as such. Though I don't have a problem with changing it, `BiomeLayers` is still an effective name.